### ISSUE TITLE
Update documentation: Producer configuration

### ISSUE THREAD

**Issue by jexp** [neo4j-contrib/neo4j-streams#66]
Add missing config docs, e.g. 

`kafka.routing.relationships.<TOPIC>=PATTERN`

is missing.

We also have no examples. Something like this (need to test):

```
kafka.routing.nodes.MasterData=Person{*};Address{street, zip, city};Profile{-description}
kafka.routing.relationships.MasterData=LIVES_AT;HAS_PROFILE

```

**Comment by conker84** [comment]
Fixed in #71 see:
https://github.com/neo4j-contrib/neo4j-streams/blob/master/doc/asciidoc/producer/patterns.adoc

**Comment by conker84** [comment]
I'll close this. Feel free to reopen if do you think something is missing
